pytorch_lightning/loggers/base.py

```diff
@@ -335,6 +335,13 @@ def save_dir(self) -> Optional[str]:
         """
         return None
 
+    @property
+    def folder_separator(self):
```

Comment:
I propose the name group_separator or grouping separator to distinguish it from a filesystem folder structure.